torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5313,6 +5351,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCPU=all_types_and_complex_and(torch.float16, torch.bfloat16),
            dtypesIfCUDA=floating_and_complex_types_and(torch.float16, *[torch.bfloat16] if CUDA11OrLater else []),
            assert_autodiffed=True,
+           supports_forward_ad=True,
```

Comment:
This should be in the last PR

Reply:
No, the formula for "mm" is added in this PR line 930 of derivatives.yaml

Reply:
thanks for pointing that out, I missed it